wizard/src/types.ts

```diff
@@ -41,9 +41,9 @@ export interface MappingVal {
   // Column that holds the mapping values. Should be set if type is
   // MappingType.COLUMN
   column?: Column;
-  // If column is set, the values in the column correspond to this property in
-  // the KG
-  valueProperty?: string;
+  // When MappedThing is PLACE, the values correspond to this type and property
```

Comment:
I don't think it's because of the TypeProperty that we're restricting it to PLACE, but it's because we only want to set this field when it's the MappingVal for a PLACE? I think also, we are only giving user TypeProperty options when they've mapped a column to PLACE

Reply:
Agreed.

Reply:
wait, agreed that it should be restricted to PLACE or shouldn't be restricted to PLACE?